docs/Current/01-search-server/meilisearch.md

```diff
@@ -0,0 +1,70 @@
+# Meili search
+
+## Background
+
+A from scratch engine, not based on Solr/Lucene.
+Built in Rust. Fully open source
+
+## Pricing
+
+Self hosted, or
+
+$1200/month base price for 10M queries and docs with dedicated system.
+Shared system not interested due to GDPR concerns IMO. Can still be placed in EU.
+Unsure if "dedicated" is actually a physical machine.
+
+
+## Score card
+### Requirements
+> 1. be fast and scalable (i.e. we must be able to start small and grow with minimal concern for performance issues when scaled up)
+
+Not distributed, so only vertically scalable (unlikely problem in the near future)
+
+> 2. have a billing model we can understand and control
+	* Think that "search as you type" can produce *many* queries
+	* Preferably something that does not scale linearly with number of requests
+
+Base cost + number of reqs in hosted version.
+
+> 3. be a mature product that we have reason to believe will be maintained for the next 3-5 years at least
+
+Series A in 2021. 1.0 release in early 2023. Currently little activity in the repo.
+Somewhat unclear how stable the company is.
+
+> 4. be able to comply with GDPR (EU hosting at a minimum)
+
+Can self host or give let them do the hosting. 
+Pricing is steep 0$ -> 1200$ -> Custom
+
+> 5. be reasonably easy to use for a developer (no need a math degree to understand what's going on)
+
+Yes, many integrations/SDKs provided
+
+3/5 Points
+
+### Wishlist
+
+* Permissions - Scoped API keys https://typesense.org/docs/0.24.0/api/api-keys.html#create-an-api-key
+* Multiple languages (stemming, stop words, etc) - Yes
+* Fuzzy search - Yes 
+* Faceting - Yes
+* Ability to import large documents (300 kb+) - Yes, 2GB
+* Ability to import many documents (1M+) - Yes, limited by disk
+* Allow weights/boosting - No - They seems to have some advanced "relevancy config"
+* Match highlighting - Yes https://rubygems.meilisearch.com/?q=bundler
+
+7/8 Points
+
+## Technical test
+
+Worked well. Responses are full documents.
+Speed is nothing amazing. Don't see benefit over Elastic.
+
+### Issues
+
+* IDs are not autogenerated. Not a big issue just seems odd
+
+
+## Summary
+
+Somewhat meh experience. Don't feel like it adds anything compared to Elastic.
```

Comment:
Is there no such document for Elasticsearch?

Reply:
Right, forgot to write that one, will add.